.gitlab-ci.yml

```diff
@@ -51,3 +52,27 @@ notify_failure:
       when: never
     - if: '$CI_COMMIT_BRANCH == "main"'
       when: on_failure
+
+
+#Coverity Scan for versatile-data-kit repository
+coverity-scan:
+  stage: coverity-scan
+  image: maven:3.6.3-jdk-11
+  script:
+    - wget https://scan.coverity.com/download/linux64 --post-data "token=$COVERITY_SCAN_TOKEN&project=$COVERITY_SCAN_PROJECT_NAME" -O /tmp/coverity_tool.tgz
+    - tar xf /tmp/coverity_tool.tgz
+    - cov-analysis-linux64-*/bin/cov-configure --python
+    - cov-analysis-linux64-*/bin/cov-build --dir cov-int --no-command --fs-capture-search projects/vdk-plugins
+    - cov-analysis-linux64-*/bin/cov-build --dir cov-int --no-command --fs-capture-search projects/vdk-core
+    - cov-analysis-linux64-*/bin/cov-build --dir cov-int --no-command --fs-capture-search projects/vdk-control-cli
+    - cov-analysis-linux64-*/bin/cov-build --dir cov-int --no-command --fs-capture-search projects/control-service
+    - tar czf cov-int.tar.gz cov-int
+    - curl
+      --form token=$COVERITY_SCAN_TOKEN
+      --form email=$COVERITY_SCAN_EMAIL
+      --form file=@cov-int.tar.gz
+      --form description="Coverity scan for vdk"
+      https://scan.coverity.com/builds?project=$COVERITY_SCAN_PROJECT_NAME
```

Comment:
I don't actually think you need the variable here I would actually prefer if it was just hardcoded. easier to just click.

Reply:
I have updated the file to use the project name instead of the variable name.
NOTE: One is required to sign in to view the coverity results.